app/packs/src/components/common/CommentModal.js

```diff
@@ -167,18 +166,20 @@ export default class CommentModal extends Component {
     if (sectionComments?.length > 0) {
       return sectionComments.map((comment) => (
         <tr key={comment.id}>
-          <td width="20%">
+          <td className='w-20'>
```

Comment:
In my setup eslint keeps mentioning using double quotes for HTML/JSX attributes and single quotes only for JS strings.